### PR TITLE
Enable checkAncestors when computing Baseline status

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

File: features/conic-gradients.yml.dist
Comment (foolip) [inline features/conic-gradients.yml.dist:5]: This will be fixed by https://github.com/mdn/browser-compat-data/pull/23603.

File: features/clipboard-unsanitized-formats.yml.dist
Comment (foolip) [inline features/clipboard-unsanitized-formats.yml.dist:7]: The cause of this is here:

https://github.com/mdn/browser-compat-data/blob/ebb58c142ff3f1dcade8857de96d16fb8e2f80f1/api/Clipboard.json#L46-L71

The notes don't actually say what isn't supported...

File: features/js-modules-shared-workers.yml.dist
Comment (foolip) [inline features/js-modules-shared-workers.yml.dist:11]: The 16 comes from https://github.com/mdn/browser-compat-data/blob/ebb58c142ff3f1dcade8857de96d16fb8e2f80f1/api/SharedWorker.json#L35.

It seems like this shouldn't pass BCD's lints, there claim that ES modules is supported in Safari 15 is at odds with the data for `SharedWorker`.

File: features/clipboard-unsanitized-formats.yml.dist
Comment (ddbeck) [reply features/clipboard-unsanitized-formats.yml.dist:7]: Hmm, yeah, that's iffy data. Two thoughts here:

- The partial seems to be the requirement to have the (non-standard?) `clipboard-read` permission. I can't find that permission specified anywhere, which makes it hard to understand.

- Those media types should be subfeatures of `read` instead of notes. Probably something like `text/plain` and `image/png` ought to be in `compute_from` list?

File: features/conic-gradients.yml.dist
Comment (ddbeck) [reply features/conic-gradients.yml.dist:5]: Should we site on this, until it's closer to the release? Or should we just avoid doing a release until this is fixed upstream?

File: features/js-modules-shared-workers.yml.dist
Comment (ddbeck) [reply features/js-modules-shared-workers.yml.dist:11]: I filed https://github.com/mdn/browser-compat-data/issues/23624

File: features/conic-gradients.yml.dist
Comment (foolip) [reply features/conic-gradients.yml.dist:5]: Let's wait for the next BCD release before merging this PR.

File: features/js-modules-shared-workers.yml.dist
Comment (foolip) [reply features/js-modules-shared-workers.yml.dist:11]: I see. I've tried to figure out if TypeScript can constrain strings to a specific format but don't know the answer. How about we just make `yamlEntries` throw an error if an identifier isn't in the form we like?

**Review by ddbeck — APPROVED** [review]
With https://github.com/web-platform-dx/web-features/pull/1332 merged, I think this is ready for a rebase and merge.

**Comment by ddbeck** [comment]
OK, how about this: we turn off the ancestor check, but add an issue (here? on BCD?) saying we ought to do something about the keys we find via `find-troublesome-ancestors.ts`?

Seems weird that we should know about the problem but not do anything about it.

**Comment by foolip** [comment]
Let's do this in https://github.com/web-platform-dx/web-features/pull/1485 instead.

**Pull request closed without merging**
